apidef/oas/operation_test.go

```diff
@@ -27,6 +27,7 @@ func TestOAS_PathsAndOperations(t *testing.T) {
 	var operation Operation
 	Fill(t, &operation, 0)
 	operation.ValidateRequest = nil // This one also fills native part, let's skip it for this test.
+	operation.TransformRequestBody.Path = ""
```

Comment:
I couldn't understand this part.

Reply:
Missed to add a comment there. If `path` and `body` are present in transform body config, `body` would take precedence. Since `OAS.Fill()` adds values for all, I've set this to be empty and added detailed test separately.